oap-stdlib/src/main/java/oap/application/KernelHelper.java

```diff
@@ -171,4 +176,12 @@ public static void restoreThreadName() {
         }
 
     }
+
+    record Parameter(Object value, boolean ignoreCast) {
+    }
+
+    static class Parameters {
```

Comment:
Where we can find the list of all supported `configurationParameters` and `serviceParameters`? Or they are declared in config files itself?

Reply:
Parameters -> ServiceConfigurationParameters
Parameter -> ServiceConfigurationParameter